python/looker_sdk/sdk/api31/methods.py

```diff
@@ -3682,7 +3682,7 @@ def fetch_integration_form(
     # POST /data_actions/form -> models.DataActionForm
     def fetch_remote_data_action_form(
         self,
-        body: MutableMapping[str, str],
+        body: MutableMapping[str, Any],
```

Comment:
why don't the other languages have these `methods` updates to `fetch_remote_data_action_form` and `query_task_multi_results` ? I see that TS still uses `IDictionary<string>` in these cases.

Reply:
if this is correct then we need [this PR](https://github.com/looker-open-source/sdk-codegen/pull/233)

Reply:
As I mentioned above, TS doesn't need this because the JS run-time tolerates the ambiguity when parsing our JSON payloads (much like Ruby and untyped Python do).

Kotlin and Swift *should* have these updates as well. Maybe I'm misunderstanding?

```kotlin
  fun fetch_remote_data_action_form(
    /**
     * @param {Map<String,Any>} body
     */
    body: Map<String,Any>) : SDKResponse {
      return this.post<DataActionForm>("/data_actions/form", mapOf(), body)
  }
```

```swift
    func fetch_remote_data_action_form(
        /**
         * @param {StringDictionary<AnyCodable>} body
         */
        _ body: StringDictionary<AnyCodable>,
        options: ITransportSettings? = nil
    ) -> SDKResponse<DataActionForm, SDKError> {
        let result: SDKResponse<DataActionForm, SDKError> = self.post("/data_actions/form", nil, try! self.encode(body), options)
        return result
    }
```

Reply:
ah, I didn't see the changes in this PR for kotlin/swift so I assumed they were different but now I see they were changed in an earlier PR.

That makes sense for TS except that I'd suspect that the typing could potentially be wrong and perhaps require casting (`as`)?

Reply:
All Typescript type checking is gone at run-time, so if JS doesn't complain about the JSON payload format, it's not going to complain.

Because this is a hack workaround anyway (we need to fix our spec then back out this demotion of `string` to `any` for the generation of the SDKs that care about the payload type at run-time) I don't think doing the same hack for Typescript is needed.